### PR TITLE
stats: use atomic ints instead lock

### PR DESCRIPTION
**Pull request by sakateka**

### CODE REVIEW

File: combainer/stats.go [hunk 1]
```diff
@@ -1,52 +1,40 @@
...
+	atomic.StoreInt64(&cs.last, time.Now().Unix())
 }
 
-func (cs *clientStats) GetStats() (info *StatInfo) {
-	cs.RLock()
-	defer cs.RUnlock()
-
-	info = &StatInfo{
+func (cs *clientStats) GetStats() *StatInfo {
+	return &StatInfo{
```
Comment (noxiouz) [inline combainer/stats.go]: atomic.Load for read

Comment (sakateka) [inline combainer/stats.go]: This is not necessarily the race there will not be, but there is a chance to get the old data, and it's not bad.

Comment (noxiouz) [inline combainer/stats.go]: -race

Comment (sakateka) [inline combainer/stats.go]: fix test, and it really cause race)

**Pull request merged**